packages/pigeon/platform_tests/test_plugin/windows/pigeon/core_tests.gen.cpp

```diff
@@ -1447,82 +1430,405 @@ const flutter::StandardMessageCodec& FlutterIntegrationCoreApi::GetCodec() {
+        const auto& encodable_return_value = *response;
+        const auto& return_value = std::get<bool>(encodable_return_value);
+        on_success(return_value);
+      });
+}
+void FlutterIntegrationCoreApi::EchoInt(
+    int64_t an_int_arg, std::function<void(int64_t)>&& on_success,
+    std::function<void(const FlutterError&)>&& on_error) {
+  auto channel = std::make_unique<flutter::BasicMessageChannel<>>(
+      binary_messenger_, "dev.flutter.pigeon.FlutterIntegrationCoreApi.echoInt",
+      &GetCodec());
+  flutter::EncodableValue encoded_api_arguments =
+      flutter::EncodableValue(flutter::EncodableList{
+          flutter::EncodableValue(an_int_arg),
+      });
+  channel->Send(
+      encoded_api_arguments,
+      [on_success, on_error](const uint8_t* reply, size_t reply_size) {
+        std::unique_ptr<flutter::EncodableValue> response =
+            GetCodec().DecodeMessage(reply, reply_size);
+        const auto& encodable_return_value = *response;
+        const int64_t return_value = encodable_return_value.LongValue();
+        on_success(return_value);
+      });
+}
+void FlutterIntegrationCoreApi::EchoDouble(
+    double a_double_arg, std::function<void(double)>&& on_success,
+    std::function<void(const FlutterError&)>&& on_error) {
+  auto channel = std::make_unique<flutter::BasicMessageChannel<>>(
+      binary_messenger_,
+      "dev.flutter.pigeon.FlutterIntegrationCoreApi.echoDouble", &GetCodec());
+  flutter::EncodableValue encoded_api_arguments =
+      flutter::EncodableValue(flutter::EncodableList{
+          flutter::EncodableValue(a_double_arg),
+      });
+  channel->Send(
+      encoded_api_arguments,
+      [on_success, on_error](const uint8_t* reply, size_t reply_size) {
+        std::unique_ptr<flutter::EncodableValue> response =
+            GetCodec().DecodeMessage(reply, reply_size);
+        const auto& encodable_return_value = *response;
+        const auto& return_value = std::get<double>(encodable_return_value);
+        on_success(return_value);
+      });
+}
+void FlutterIntegrationCoreApi::EchoString(
     const std::string& a_string_arg,
-    std::function<void(const std::string&)>&& callback) {
-  auto channel =
-      std::make_unique<flutter::BasicMessageChannel<flutter::EncodableValue>>(
-          binary_messenger_,
-          "dev.flutter.pigeon.FlutterIntegrationCoreApi.echoString",
-          &GetCodec());
+    std::function<void(const std::string&)>&& on_success,
+    std::function<void(const FlutterError&)>&& on_error) {
+  auto channel = std::make_unique<flutter::BasicMessageChannel<>>(
+      binary_messenger_,
+      "dev.flutter.pigeon.FlutterIntegrationCoreApi.echoString", &GetCodec());
+  flutter::EncodableValue encoded_api_arguments =
+      flutter::EncodableValue(flutter::EncodableList{
+          flutter::EncodableValue(a_string_arg),
+      });
   channel->Send(
-      flutter::EncodableList{flutter::CustomEncodableValue(a_string_arg)},
```

Comment:
This is showing the main fix: this is the old `EchoString` implementation, and it was trying to use `CustomEncodableValue`, which is only for the custom data classes, so won't actually work. Now it's an `EncodableValue` as expected.